docs/config/lua/window/toast_notification.md

```diff
@@ -11,7 +11,9 @@ An optional *timeout* parameter can be provided; if so, it specifies how long
 the notification will remain prominently displayed in milliseconds.  To specify
 a timeout without specifying a url, set the url parameter to `nil`.  The timeout
 you specify may not be respected by the system, particularly in X11/Wayland
```

Comment:
The docs already say that the timeout may not be respected up here.
Rather than making these document current behaviors of specific versions
of Windows, and referencing something that users cannot change in the code,
I'd rather that this just be a generic statement that you cannot rely on the timeout
being supported by any system.

Reply:
Right this is reasonable, I just thought it was worth mentioning somewhere for those that might come along and face a similar issue of desiring a shorter timeout. Killer termina emulator non-the-less!